src/screens/LoginScreen/index.tsx

```diff
@@ -18,9 +18,10 @@ import { useLoginMutation } from '@src/operations/mutations/login';
 import { userState } from '@src/recoils';
 import { AuthStackParamList, RootStackParamList } from '@src/types/navigation';
 
+import useSetRecoilStates from '../../hooks/useSetRecoilStates';
```

Comment:
이 친구는 형제 파일이 아니니까 `@src/hooks` 형식으로 import 하는게 좋을 것 같아요〰️

```suggestion
import useSetRecoilStates from '@src/hooks/useSetRecoilStates';
```

Reply:
654b024a831fa7a1e799d0873a825917d5634ea3 수정했습니다 :)